registrations/models.py

```diff
@@ -119,16 +120,35 @@ def fire_unique_operator_metric(sender, instance, created, **kwargs):
 
 @receiver(post_save, sender=Registration)
 def fire_message_type_metric(sender, instance, created, **kwargs):
-    """Fires a metric for each message type of each subscription."""
+    """
+    Fires a metric for each message type of each registration.
+
+    Fires both a `sum` metric with 1.0 for each registration, as well as a
+    `last` metric with the total amount of registrations for that type.
+    """
     from .tasks import fire_metric, is_valid_msg_type
-    if (created and instance.data and instance.data['msg_type'] and
+    if (created and instance.data and instance.data.get('msg_type') and
             is_valid_msg_type(instance.data['msg_type'])):
+        msg_type = instance.data['msg_type']
         fire_metric.apply_async(kwargs={
-            "metric_name": 'registrations.msg_type.%s.sum' % (
-                instance.data['msg_type']),
+            "metric_name": 'registrations.msg_type.%s.sum' % msg_type,
             "metric_value": 1.0,
         })
 
+        total_key = 'registrations.msg_type.%s.last' % msg_type
+        total = cache.get(total_key)
+        if total is None:
+            total = Registration.objects.filter(
+                data__msg_type=msg_type).count()
```

Comment:
I'm worried about the performance implications of doing a `count` across all registrations every 10 minutes, postgres isn't very good at counting, and doing it this way will only have us doing `count` for the first registration, and then we never have to do the `count` unless we lose the cache.

The metrics API will automatically aggregate things, so we don't need to worry about filling up storage space in the metrics DB.